modules/punctuation/punctuation.cpp

```diff
@@ -395,4 +398,54 @@ Punctuation::getPunctuation(const std::string &language, uint32_t unicode) {
     return iter->second.getPunctuation(unicode);
 }
 
+const fcitx::Configuration *
+Punctuation::getSubConfig(const std::string &path) const {
+    if (path == "punctuationmap-zh_CN") {
```

Comment:
It can be any language code. Or at least you should make it accept all three variant that supported by fcitx (zh_CN/zh_TW/zh_HK)